dist/IO/IO.xs

```diff
@@ -553,18 +553,24 @@ PPCODE:
     if (!ix && GIMME_V != G_LIST)
         Perl_croak(aTHX_ "Can't call $io->getlines in a scalar context, use $io->getline");
     Zero(&myop, 1, UNOP);
+#if defined(PERL_VERSION_GE) && PERL_VERSION_GE(5,39,5)
```

Comment:
`ppport.h` defines `PERL_VERSION_GE` so there's no need to check this here and below.